common/src/test/java/com/microsoft/identity/common/BrokerOAuth2TokenCacheTest.java

```diff
@@ -289,15 +289,17 @@ private ISharedPreferencesFileManager getAppUidFileManager(final Context context
         return SharedPreferencesFileManager.getSharedPreferences(
                 context,
                 getBrokerUidSequesteredFilename(appUid),
-                new StorageHelper(context)
+                -1,
```

Comment:
This appears to be missing or paving over changes I did in the following PR
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1444/files#diff-d8fd7f57e0a3e84b81cf6959b128979754729ab74636f5b67f4d5fa0d7fb7b2d

Where `-1` is no longer being passed